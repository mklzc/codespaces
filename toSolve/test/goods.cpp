#include <bits/stdc++.h>
#define rep(var, st, ed) for(int var = st; var <= ed; var++)
#define fin freopen("in", "r", stdin)
#define fout freopen("out", "w", stdout)
namespace IO{
    template <typename T>
    inline void read(T &x)
    {
        x = 0;
        T w = 1;
        char ch = getchar();
        while (ch < '0' || ch > '9') { if (ch == '-') w = -1;ch = getchar();}
        while (ch >= '0' && ch <= '9') x = x * 10 + ch - 48, ch = getchar();
        x = x * w;
    }
    template <typename T, typename... Args>
    inline void read(T &x, Args&... args) {read(x);read(args...);}
};
using IO::read;
using LL = long long;
const int Maxn = 605;
LL a[Maxn], L, sum = 0;
int m;
LL cnt(int Weight)
{
    return a[Weight + m + 1];
}
void dfs()
{
}
int main()
{
    read(m, L);
    for (int i = 1, j = -m; i <= 2 * m + 1; i++, j++)
        read(a[i]), sum += j * a[i];
	return 0;
}