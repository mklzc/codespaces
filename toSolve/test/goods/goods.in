30 24025640327107
41591927263 15 16 21 8 10 802906198561 380904647313 436706099173 10 5 8 184580419211 64155825974 1 304261600343 869114876645 781473561479 988004422990 46034640463 10 11 450102275405 28 3 505693515891 18 16 2 7 976865660464 29 0 720678795261 472008146716 9459665058 784116611800 404878641878 19 19 18 39189462270 504197992188 851866899162 58566351389 167797235436 1 24 7 14 3 23 13 20 342142269354 2 783867015229 3 436985564486 962816459098 380964282532
