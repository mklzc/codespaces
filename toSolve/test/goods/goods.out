13751931063415
